src/lib/utils/inventoryUtils.ts

```diff
@@ -1,4 +1,11 @@
+import { InventoryRecord } from "../airtable/interface";
+
 // TODO @wangannie replace with robust id generation
 export const generateOfflineInventoryId = (): string => {
     return (Math.floor(Math.random() * 1000)).toString();
 }
+
+// Calculate when an inventory record was last updated
+export const lastUpdated = (inventory: InventoryRecord) => {
```

Comment:
NIT: generally we want verb names. `getLastUpdated` or `calculateLastUpdated` or something along those lines?